policyengine_us/variables/gov/states/sc/tax/income/sc_state_tax_addback.py

```diff
@@ -0,0 +1,51 @@
+from policyengine_us.model_api import *
+
+
+class sc_state_tax_addback(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "South Carolina State Tax addback"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://dor.sc.gov/forms-site/Forms/SC1040_2022.pdf",
+        "https://dor.sc.gov/forms-site/Forms/SC1040inst_2022.pdf#page=2",
+    )
+    defined_for = StateCode.SC
+
+    def formula(tax_unit, period, parameters):
+        p_us = parameters(period).gov.irs.deductions
+        itm_deds = [
+            deduction
+            for deduction in p_us.itemized_deductions
+            if deduction not in ["salt_deduction"]
+        ]
+        us_itemizing = tax_unit("tax_unit_itemizes", period)
+        standard_deduction = tax_unit("standard_deduction", period)
+        filing_status = tax_unit("filing_status", period)
+        eligible = filing_status != filing_status.possible_values.SEPARATE
+        # line 1
+        federal_itemized_deduction = (
+            add(tax_unit, period, itm_deds) * us_itemizing
+        )
+        # line 2
+        federal_standard_deduction = standard_deduction * eligible
+        # line 3
+        less_itm_amount = max_(
+            0, federal_itemized_deduction - federal_standard_deduction
+        )
+        # line 4
+        salt = tax_unit("state_and_local_sales_or_income_tax", period)
+        # line 5
+        capped_property_taxes = min_(
```

Comment:
I think line 5 is just a calculation where the filer takes the property taxes and subtracts them from the federal cap of 10,000 (5,000 if separate)

Reply:
the property taxes themselves in this condition are not capped